RecoTracker/IterativeTracking/python/PixelLessStep_cff.py

```diff
@@ -288,14 +292,19 @@
 
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(pixelLessStep, TrackMVAClassifierDetached.clone(
```

Comment:
Changed back to being consistent.

I can also confirm that the training script used for the BDTs in this and similarly affected pixelPairStep were trained as detached instead of prompt versions. They are now changed back to the prompt versions here, as that is how they've (mistakenly) been for a long time now it seems. Let me know if I should correct this cosmetic error now or in some other context.